lib/database/schemas.js

```diff
@@ -58,6 +58,10 @@ const content = [
       update: 'CASCADE',
     },
   },
+  {
+    name: 'workflows',
+    type: 'jsonb',
+  },
```

Comment:
yes. comments/audit trail.

Object created for that entry: https://github.com/scottnath/punchcard/blob/f247101e6d96eb0553804c8c56d5930c939f3046/lib/routes/workflows.js#L163

`workflows` will have an array of those objects.